packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -863,6 +879,158 @@ describe('serverless cluster', () => {
 
 
   });
+
+  test('can create a Serverless cluster without vpc', () => {
+    // GIVEN
+    const stack = testStack();
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
+      Engine: 'aurora-postgresql',
+      DBClusterParameterGroupName: 'default.aurora-postgresql10',
+      EngineMode: 'serverless',
+      MasterUsername: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:username::}}',
+          ],
+        ],
+      },
+      MasterUserPassword: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:password::}}',
+          ],
+        ],
+      },
+    });
+
+
+  });
+
+  test('can create a Serverless cluster without vpc but with imported security group', () => {
+    // GIVEN
+    const stack = testStack();
+    const sg = ec2.SecurityGroup.fromSecurityGroupId(stack, 'SG', 'SecurityGroupId12345');
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      securityGroups: [sg],
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
```

Comment:
Same comment here (this assertion is too broad for this test). Just check `VpcSecurityGroupIds`.

Reply:
👍🏻